### PR TITLE
Update `terraform-fmt-command.yml` GitHub Actions workflow

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Update `terraform-fmt-command.yml` GitHub Actions workflow

## why
* Format terraform code in `examples/complete` as well

### CODE REVIEW

File: .github/workflows/terraform-fmt-command.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ jobs:
           export TERRAFORM_VERSION=0.12.19
           make terraform/install
           terraform fmt
+          cd ./examples/complete 2>/dev/null
```
Comment (osterman) [inline .github/workflows/terraform-fmt-command.yml]: it would be better to do something like
```
find . -type f -name '*.tf' -exec terraform fmt {} \;
```

Comment (osterman) [reply .github/workflows/terraform-fmt-command.yml]: actually, even better:

```
terraform fmt  -recursive
```

File: .github/workflows/terraform-fmt-command.yml [hunk 2]
```diff
@@ -22,6 +22,8 @@ jobs:
...
+          terraform fmt 2>/dev/null
```
Comment (osterman) [inline .github/workflows/terraform-fmt-command.yml]: why hide `stderr`?

File: .github/workflows/terraform-fmt-command.yml [hunk 3]
```diff
@@ -22,6 +22,8 @@ jobs:
           export TERRAFORM_VERSION=0.12.19
```
Comment (osterman) [inline .github/workflows/terraform-fmt-command.yml]:
```suggestion
```

File: .github/workflows/terraform-fmt-command.yml [hunk 4]
```diff
@@ -22,6 +22,8 @@ jobs:
...
           make terraform/install
```
Comment (osterman) [inline .github/workflows/terraform-fmt-command.yml]:
```suggestion
          make terraform/install TERRAFORM_VERSION=0.12.19
```

File: .github/workflows/terraform-fmt-command.yml (hunk 1 above)
Comment (aknysh) [reply .github/workflows/terraform-fmt-command.yml]: `terraform fmt  -recursive` is the best

File: .github/workflows/terraform-fmt-command.yml (hunk 2 above)
Comment (aknysh) [reply .github/workflows/terraform-fmt-command.yml]: with `terraform fmt  -recursive` not necessary

**Pull request merged**